src/main/resources/embedded_ldap.properties

```diff
@@ -1,6 +1,5 @@
-spring.ldap.embedded.base-dn=dc=memorynotfound,dc=com
+spring.ldap.embedded.base-dn=dc=moj,dc=com
```

Comment:
Make in-memory LDAP same base as default real LDAP removing need for extra properties during testing etc